test/helper.rb

```diff
@@ -70,6 +74,34 @@ def run(*)
 
 module TestSkips
 
+  # finds bad signals, value is an array of symbols
+  BAD_SIGNAL_LIST = begin
```

Comment:
Not a huge fan of this, what's wrong with Signal.list?

Reply:
I appreciate the idea, but let's just remove :TERM from the list if on Windows, this spawning-a-process thing feels like the nuclear option.

Reply:
Also, if :TERM doesnt work on windows, why do tests that use not fail? e.g. `test_term_closes_listeners` or `test_term_suppress`

Reply:
Of of course, because you can't fork on windows. D'oh!

Reply:
> let's just remove :TERM from the list if on Windows, this spawning-a-process thing feels like the nuclear option

Understandable.

I really hate defining by OS, as opposed to actually checking for functionality.  Maybe it (`:TERM`) could change in a year or something.  Probably unlikely...